README.md

```diff
@@ -42,7 +43,7 @@ Auto completion plugin for nvim.
 - Better fuzzy matching algorithm
   - `gu` can be matched `get_user`
   - `fmodify` can be matched `fnamemodify`
-  - See [detailed explanation](./lua/compe/matcher.lua:L57) if you interest it
+  - See [detailed explanation](./lua/compe/matcher.lua:L57) if you're interested
```

Comment:
Perhaps `See … for details` or `for implementation( details))` would be better?

Reply:
agree